app/api/chemotion/molecule_api.rb

```diff
@@ -41,11 +42,9 @@ class MoleculeAPI < Grape::API
           unless molecule
             molfile = babel_info[:molfile] if babel_info
             begin
-              rw_mol = RDKitChem::RWMol.mol_from_smiles(smiles)
-              rd_mol = rw_mol.mol_to_mol_block unless rw_mol.nil?
+              rd_mol = RdkitExtensionService.smiles_to_ctab(smiles)
             rescue StandardError => e
               Rails.logger.error ["with smiles: #{smiles}", e.message, *e.backtrace].join($INPUT_RECORD_SEPARATOR)
-              rd_mol = rw_mol.mol_to_mol_block(true, -1, false) unless rw_mol.nil?
```

Comment:
is there a test case for this? - not sure what was the line about?

Reply:
it is preventing to kekulize  
but we don't have original cases

also the rdkit cartridge packed moreup to date rdkit function  while the rdkit-chem d need an upgrade